routes/auth.js

```diff
@@ -0,0 +1,42 @@
+const router = require('express').Router();
+const { body } = require('express-validator');
+
+const authController = require('../controllers/auth');
+const User = require('../models/user');
+
+router.post(
+  '/signup',
+  [
```

Comment:
please separate this logic inside other files of middlewares. Like we can create `validateEmail` & `validatePassword`  middlewares and then here we can refer these as `[ validateEmail, validatePassword]`.